cugraph/ConnectedComponents.ipynb

```diff
@@ -0,0 +1,443 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Connected Components\n",
+    "\n",
+    "In this notebook, we will use cuGraph to compute weakly and strongly connected components of a graph and display some useful information about the resulting components.\n",
+    "\n",
+    "Notebook Credits\n",
+    "* Original Authors: Kumar Aatish\n",
+    "* Last Edit: 08/05/2019\n",
+    "\n",
+    "RAPIDS Versions: 0.9.0    \n",
+    "\n",
+    "Test Hardware\n",
+    "\n",
+    "* GV100 32G, CUDA 10,1\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "To compute WCC for a graph in cuGraph we use:\n",
+    "**cugraph.weakly_connected_components(G)**\n",
+    "\n",
+    "To compute SCC for a graph in cuGraph we use:\n",
+    "**cugraph.strongly_connected_components(G)**\n",
+    "\n",
+    "Both of these calls have identical api:  \n",
+    "\n",
+    "Input\n",
+    "* __G__: cugraph.Graph object\n",
+    "\n",
+    "Returns \n",
+    "* __df__:  a cudf.DataFrame object with two columns:\n",
+    "    * df['labels'][i]: Gives the label id of the i'th vertex\n",
+    "    * df['vertices'][i]: Gives the vertex id of the i'th vertex\n",
+    "\n",
+    "\n",
+    "\n",
+    "## cuGraph 0.9 Notice \n",
+    "cuGraph version 0.9 has some limitations:\n",
+    "* Only Int32 Vertex ID are supported\n",
+    "* Vertex numbering is assumed to start at zero\n",
+    "\n",
+    "These limitations are being addressed and will be fixed future versions.  \n",
+    "These example notebooks will illustrate how to manipulate the data so that it comforms to the current limitations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Test Data\n",
+    "We will be using the Netscience dataset :  \n",
+    "*M. E. J. Newman, Finding community structure in networks using the eigenvectors of matrices, Preprint physics/0605087 (2006)*\n",
+    "\n",
+    "Netscience Adjacency Matrix               |NetScience Strongly Connected Components\n",
+    ":-----------------------------------------|--------------------------------------------------:\n",
+    "![](./img/netscience.png \"Credit : https://www.cise.ufl.edu/research/sparse/matrices/Newman/netscience\") | ![](./img/netscience_scc.png \"Credit : https://www.cise.ufl.edu/research/sparse/matrices/Newman/netscience\")\n",
+    "\n",
+    "Matrix pictures by [cspy](http://www.cise.ufl.edu/research/sparse/CSparse/CSparse/MATLAB/CSparse/cspy.m), a MATLAB function in the [CSparse](http://www.cise.ufl.edu/research/sparse/CSparse) package.  \n",
+    "Matrix graphs by Yifan Hu, AT&T Labs Visualization Group."
```

Comment:
As far as I can tell the matrix graphs (from Yifan Hu) are the plots of the adjacency list and the strongly connected components which can be seen [here](https://www.cise.ufl.edu/research/sparse/matrices/Newman/netscience.png) and [here](https://www.cise.ufl.edu/research/sparse/matrices/Newman/netscience_scc.png).
And I think that the line about the matrix pictures (cspy) can be removed because I think it refers to the graph plot seen [here](http://yifanhu.net/GALLERY/GRAPHS/GIF_SMALL/Newman@netscience.gif) which is not being used in the notebook.